ccm_web/client/src/pages/AddUMUsers.tsx

```diff
@@ -0,0 +1,450 @@
+        const parts = line.split(',')
+        if (parts.length !== 2) {
+          errors.push({ rowNumber: i + 2, message: 'Invalid column count' })
+        } else {
+          if (!isValidRole(parts[0])) {
+            errors.push({ rowNumber: i + 2, message: `Invalid ${USER_ROLE_TEXT.toUpperCase()} '${parts[0]}'` })
+          } else if (!isValidLoginID(parts[1])) {
+            errors.push({ rowNumber: i + 2, message: `Invalid ${USER_ID_TEXT.toUpperCase()} '${parts[1]}'` })
+          } else {
+            enrollments.push({ rowNumber: i + 2, loginID: parts[1], role: parts[0] })
+          }
+        }
+      })
+
+      if (errors.length === 0) {
+        handleParseSuccess(enrollments)
+      } else {
+        handleParseFailure(errors)
+      }
+    }).catch(e => {
+      // TODO Not sure how to produce this error in real life
+      console.log('error parsing file')
+    })
+  }
+
+  const getSelectContent = (): JSX.Element => {
+    if (getCanvasSectionDataError === undefined) {
+      return (
+        <>
+          <div className={classes.createSetctionWidget}><CreateSectionWidget {...props} onSectionCreated={sectionCreated}/></div>
+          <Typography variant='subtitle1'>Or select one available section to add users</Typography>
+          <div className={classes.sectionSelectionContainer}>
+            <SectionSelectorWidget height={400} multiSelect={false} sections={sections !== undefined ? sections : []} selectedSections={selectedSections} selectionUpdated={setSelectedSections}></SectionSelectorWidget>
+            <div>
+              <Button className={classes.sectionSelectButton} variant='contained' color='primary' disabled={selectedSections.length === 0} onClick={() => { setActiveStep(activeStep + 1) }}>Select</Button>
+            </div>
+            <Backdrop className={classes.backdrop} open={isExistingSectionsLoading}>
+              <Grid container>
+                <Grid item xs={12}>
+                  <CircularProgress color="inherit" />
+                </Grid>
+                <Grid item xs={12}>
+                  Loading sections
+                </Grid>
+              </Grid>
+            </Backdrop>
+          </div>
+        </>
+      )
+    } else {
+      return (
+        <Paper className={classes.sectionLoadError} role='alert'>
+          <Typography>Error loading sections</Typography>
+          <ErrorIcon className={classes.sectionLoadErrorIcon} fontSize='large'/>
+        </Paper>
+      )
+    }
+  }
+
+  const renderUploadHeader = (): JSX.Element => {
+    const fileData =
+`${USER_ROLE_TEXT.toUpperCase()},${USER_ID_TEXT.toUpperCase()}
+student,studenta
+teacher,usera
+ta,userb
+observer,userc
+designer,userd`
+    const fileDownloadHeaderProps: ExampleFileDownloadHeaderProps = {
+      bodyText: `Your file should include the user's ${USER_ID_TEXT.toLocaleLowerCase()} and their ${USER_ROLE_TEXT.toLocaleLowerCase()}`,
+      fileData: fileData,
+      fileName: 'bulk_enroll.csv',
```

Comment:
Nip-pick: can we exclusively state that `bulk_um_enroll.csv` since we have one more feature (bulk enroll non-umich user) so to make it clear with they download sample csv file